### PR TITLE
Remove unnecessary files

### PR DESCRIPTION
**Pull request by zidder**
Next step would be fixing lint errors, and making the code more readable.

@a-arakelian , @narek-mkrtumyan Any suggestions about the `Segment` class?

### CODE REVIEW

File: lib/aramanim.py [hunk 1]
```diff
@@ -4,46 +4,15 @@
...
-XELATEX_preamble = r"""
-\usepackage{tikz}
-\usepackage{fontspec}
-\setmainfont{DejaVu Serif}
-\usepackage{amsfonts,amssymb,amsthm,mathtools}
-\usepackage{amsmath}
-\usepackage{upgreek}
-\usepackage{mathrsfs}
-"""
-XELATEX = TexTemplate('xelatex', '.pdf', preamble=XELATEX_preamble)
```
Comment (a-arakelian) [inline lib/aramanim.py:41]: It seems this piece of code is being deleted here and is not duplicated anywhere. But I would like it to remain in some form. I think over time we will use the XeLaTeX compiler

**Review by a-arakelian — COMMENTED** [review]
In my opinion, everything is fine (except with XeLatex, but it is not fatal)

**Review by a-arakelian — APPROVED** [review]

File: lib/aramanim.py (hunk 1 above)
Comment (zidder) [reply lib/aramanim.py:41]: Can you look at `lib/constants.py` file, do you think we should copy this to that file? I have defined all the tex related objects there.

Comment (a-arakelian) [reply lib/aramanim.py:41]: Yes, it will be great

**Comment by zidder** [comment]
We can readd preamble later, Aram.

**Comment by a-arakelian** [comment]
> We can readd preamble later, Aram.

Ok

**Review by a-arakelian — APPROVED** [review]

**Pull request merged**